chained_filter_controller/package.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>chained_filter_controller</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
+  <maintainer email="ankur-u24@todo.todo">ankur-u24</maintainer>
+  <license>TODO: License declaration</license>
```

Comment:
please copy maintainer, license from the other packages of this repo, and add you and Bence as authors
`  <author email="bence.magyar.robotics@gmail.com">Bence Magyar</author>`

you can copy the description of the plugin xml below.